boards/pine64/0002-sunxi-configs-add-defconfig-for-pine-H64-model-B.patch

```diff
@@ -0,0 +1,48 @@
+From df5c906043659276240e62ce003217ecea545bdb Mon Sep 17 00:00:00 2001
+From: Daniel Wagenknecht <dwagenk@mailbox.org>
+Date: Wed, 8 Dec 2021 23:05:42 +0100
+Subject: [PATCH 2/4] sunxi: configs: add defconfig for pine H64 model B
+
+It almost identical to the defconfig for pine H64 model A with only
+the CONFIG_DEFUALT_DEVICE_TREE adapted.
+
+The differences between the boards model A and model B are irrelevant
+inside u-boot, but the device-tree name is relevant as default for
+selecting the correct device-tree when booting linux.
```

Comment:
(Nothing to do really)

I believe the "more correct" way to handle this would be like for the Pinephone, figure out a detectable difference between boards during early boot and setup accordingly.

I don't really know the H64. There may be nothing that can be detected for this. At least from the Linux dts, I don't really see anything.